packages/tools/client-logger/app-insights-logger/README.md

```diff
@@ -0,0 +1,11 @@
+# @fluid-tools/app-insights-logger
```

Comment:
We'll want to make sure to include our `trademark` and other boilerplate sections in this README. `api-markdown-documenter`'s README has some examples of including this via automated content generation. You'll just need to run a build from the docs directory to generate the contents once you've added the pragmas you want.